WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/CardReaderSettingsResultsControllers.swift

```diff
@@ -0,0 +1,63 @@
+import Foundation
+import Yosemite
+import protocol Storage.StorageManagerType
+
+/// Results controllers used to render the card reader settings views
+///
+final class CardReaderSettingsResultsControllers {
+    private let storageManager: StorageManagerType
+
+    private let siteID: Int64
+
+    private let instanceID = Int.random(in: 1..<10000)
```

Comment:
This seems to be unused

Reply:
Ack. Sorry about that. Fixed in https://github.com/woocommerce/woocommerce-ios/pull/6123/commits/c115fdea12c9c62a99bf5d83e64c770662d6d46f